### DISCUSSION TITLE
Removing top navigation bar in default Screen, adding Tab Bar at bottom...

### DISCUSSION THREAD

**Discussion by nick-walt** [Budibase/budibase#7188] [Help]
Hi everyone, is it possible to remove the navigation bar from the default Screen? I'm wanting to create a true SPA as mobile first and it won't look anything like a typical responsive website that has resized to mobile.

I will be replacing the default hamburger menu driven UI with a sophisticated filtering and sorting interface for a card UI design. This is to allow users to ask much more of their data without visual complexity.

I have not been able to find any reference material for implementing a Tab bar style UI that implements Tags and, later, a Search input  that will work alongside pre-defined Tags, at the bottom of the Screen. Thanks.

**Comment by melohagan** [comment]
Hi @nick-walt 

You can hide the navigation bar by clicking on the screen tab, select the screen, and toggle *Show navigation* off:
![Screenshot 2022-08-15 at 11 19 14](https://user-images.githubusercontent.com/101575380/184618658-a1c2c68a-7af5-433e-9d12-c80bd68475ca.png)

**Reply by nick-walt** [reply]
Hi @melohagan thank you. I take it then that the navigation menu bar is a part of the Budibase UI and not a part of my application?

**Reply by melohagan** [reply]
I'm not quite sure what you mean. When you turn off the navigation for a screen, it will be hidden in both the app builder, and the preview/published app.

**Reply by nick-walt** [reply]
Ah, ok, so the bar is a part of Budibase application that exists for the purpose of assisting the user in configuring their app's internal navigation? You will never see the bar in the published app because it remains a part of Budibase?

**Reply by nick-walt** [reply]
I haven't published anything, yet obviously.
